Explorer++/Explorer++/RegistrySettings.cpp

```diff
@@ -145,6 +145,9 @@ LONG Explorerplusplus::SaveSettings(void)
 		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("LargeToolbarIcons"),m_bLargeToolbarIcons);
 		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("PlayNavigationSound"),m_bPlayNavigationSound);
 
+		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("DisplayMixedFilesAndFolders"),m_bDisplayMixedGlobal);
+		NRegistrySettings::SaveDwordToRegistry(hSettingsKey,_T("SortNamesInNonLogicalOrder"),m_bSortNonLogicalGlobal);

```

Comment:
I think moving the string into constants should be done in stages in future commits, as there are some issues:

1. Should constants be in Explorer++.rc? This file gets translated, making settings not portable with languages.

2. I would prefer them in a new file, e.g. SettingConstants.h, as these are used by both registry and config.xml.

3. Except for one setting I found, registry uses AlwaysOpenNewTab while config.xml uses AlwaysOpenInNewTab - there may be others.